libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -336,6 +320,32 @@ class SRV_Channel {
     void set_override(bool b) {override_active = b;};
 };
 
+class srv_function {
```

Comment:
this is very ram expensive, about 3.5k with current functions, and will grow over time. That is too much on F405 boards for a little used feature. Almost certainly will break f103 AP_Periph PWM output too, it has very little ram
a pointer per function may be acceptable, or a bitmap of which functions have a slew rate, then a linked list or realloc list.

Reply:
moved to a linked list